rosdep/base.yaml

```diff
@@ -5535,6 +5535,11 @@ qt4-dev-tools:
   gentoo: ['dev-qt/assistant:4', 'dev-qt/linguist:4', 'dev-qt/pixeltool:4']
   opensuse: [libqt4-devel-doc]
   ubuntu: [qt4-dev-tools]
+qt4-linguist-tools:
+  debian: [qt4-linguist-tools]
+  fedora: [qt]
```

Comment:
From what I can tell, the `qt` package on fedora does not install the `lrelease` and `lupdate` binaries.  To get that, you actually need `qt-devel`.  So I'll suggest we switch to that for Fedora.